prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,72 @@
+const age = 28
+function isAdult(age){
+  if (age > 18){
+    boolVal = true
```

Comment:
you have to make sure to declare a variable when you want to create one rather than just using a variable name:

for example 
const boolVal = true

vs

boolVal = true


the reason is that if you just create a variable without declaring it, JavaScript will declare it for you using the var keyword and will automatically make it a global variable which is not what you want. All function variables should be local.